pkg/service/utils.go

```diff
@@ -33,3 +34,15 @@ func IsInternalRequest(ctx context.Context) bool {
 	b, ok := v.(bool)
 	return ok && b
 }
+
+func MatchStringToRegx(matchString, regex string) (error, bool) {
+	re, err := regexp.Compile(regex)
+	if err != nil {
```

Comment:
This can be eliminated by doing build time regex compilation.

```
var PrjNameRX = regexp.MustCompile(`^[a-zA-Z]([-a-zA-Z0-9]*[a-zA-Z0-9])?$`)
```